README.md

```diff
@@ -12,6 +12,7 @@ for deep learning matrices, such as
 - the Hessian
 - the Fisher/generalized Gauss-Newton (GGN)
 - the Monte-Carlo approximated Fisher
+- the Fisher's KFAC approximation (Kronecker-Factored Approximate Curvature)
```

Comment:
Nit: we could say Fisher/GGN's here since we also name both above (and they coincide for all losses we support with KFAC).